build/local/README.md

```diff
@@ -23,19 +36,25 @@ Build a docker image with:
 make docker-build
 ```
 
-### Push your Docker Container Image to the Artifact Registry
+## Push your Docker Container Image to the [Artifact Registry](https://cloud.google.com/artifact-registry)
 
-Set `$GCLOUD_PROJECT` and `$CONTAINER_REGISTRY` to tag and push the resulting image.
-Note: you will need an existing registry for this to work, that can be created with ``.
+Set `$GCLOUD_PROJECT` with the name of your Google Cloud project to tag and push the resulting image.
+Note: you will need an existing registry for this to work, that was created as part of the [prequisites](#prerequisite-setup-an-artifact-registry).
 
 ```
-export GCLOUD_PROJECT=my-gcp-project
-export CONTAINER_REGISTRY=custom-collectors
+export GCLOUD_PROJECT=<name of you GCP project>
```

Comment:
as a follow up it would be nice to have the makefile try to automatically parse the gcloud project like in https://github.com/GoogleCloudPlatform/opentelemetry-operator-sample/blob/86f2b4d/Makefile#L17

Reply:
That's a nice idea will try this.